### DISCUSSION TITLE
Snackbar message text alignment and snackbar height?

### DISCUSSION THREAD

**Discussion by LeoJHarris** [CommunityToolkit/Maui#378] [New Feature Discussions]
Feature request if this is not currently supported, to include properties for message text alignment and also snack bar HeightRequest?

```csharp
MessageOptions => public MessageTextAlignment MessageAlignment { get; set; }
SnackBarOptions => public double HeightRequest { get; set; }
```

Effectively something similar to the below could then be achieved:

![image](https://user-images.githubusercontent.com/5468757/164593105-7cf94327-28b9-4bc7-9028-e5d86fd7df9a.png)

I would be happy to look into implementing this but would need to get up on familiarizing with Maui first. Happy to open discussion initially.

**Comment by VladislavAntonyuk** [comment]
As for the Height you can try achieving the result by using Padding. 
As for the Text Alignment it may work only on Android and iOS/MacCatalyst. I am not sure if Windows Toast notifications support it

**Comment by CliffAgius** [comment]
Closed as answered
